core/src/main/scala/org/apache/spark/ui/jobs/AllJobsPage.scala

```diff
@@ -279,13 +289,14 @@ private[ui] class AllJobsPage(parent: JobsTab, store: AppStatusStore) extends We
 
     val schedulingMode = store.environmentInfo().sparkProperties.toMap
```

Comment:
Could you replace this duplicated code with the method call of `getSchedulingMode()`, or if we can avoid adding the new Suite as I mentioned elsewhere, we can remove `getSchedulingMode`.

Reply:
Hi @sarutak ,Thanks for reviewin，I see your thoughts, I will add the test to `UISeleniumSuite` and  replace this duplicated code with the method call of `getSchedulingMode()`.